src/booleans.js

```diff
@@ -1,61 +1,63 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !(a || b);
 };
 
 function one(a, b) {
-  // your code here
+  return (a || b) && !(a && b);
 };
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
-  // your code here
+  return a === b;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return a > b;
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  return a <= b; 
 };
 
 function isOdd(a) {
-  // your code here
+  return Boolean(a % 2);
 };
 
 function isEven(a) {
-  // your code here
+  return !(Boolean(a % 2));
 };
 
 function isSquare(a) {
-  // your code here
+  const squareRoot = Math.sqrt(a);
+  const isPerfectSquare = Math.ceil(squareRoot) === Math.floor(squareRoot);
+  return isPerfectSquare;
 };
 
 function startsWith(char, string) {
-  // your code here
+  return string.startsWith(char);
```

Comment:
This is a really unusual approach!
Calling a function recursively like this is not something you see or want to do very often, in this case it works as the `string` value is still in scope when calling the internal function and its a very simple operation, in a larger case this would be much more likely to fail. One reason for this is there is no default value for the second parameter, also what if this were required for the function to work?
An alternative would be simply
```
  return char === string[0];
```